src/types/cooperation/types/cooperation.types.ts

```diff
@@ -16,3 +16,9 @@ export type UpdateCooperationStatusParams = Partial<
 export type UpdateCooperationsNeedActionMessages = {
   newMessage: string
 } & Pick<Cooperation, '_id'>
+
+export type UpdateResourceCompletionStatusParams = {
+  id: string
+  resourceId: string
+  completionStatus: CompletionStatusEnum | undefined
```

Comment:
Check, please, if it works like this:

```suggestion
  completionStatus?: CompletionStatusEnum
```

Reply:
fixed

Reply:
done